subsys/sd/mmc.c

```diff
@@ -111,9 +111,9 @@ int mmc_card_init(struct sd_card *card)
 	    card->host_props.host_caps.vol_300_support) {
 		ocr_arg |= MMC_OCR_VDD27_36FLAG;
 	}
-	 /* Modern SDHC always at least supports 512 byte block sizes,
-	  * which is enough to support sectors
-	  */
+	/* Modern SDHC always at least supports 512 byte block sizes,
```

Comment:
These changes to `mmc_card_init`, beyond the change to `card_ext_csd`, should be squashed into the first commit implementing this function. Sorry for missing this earlier